### ISSUE TITLE
HelloWorld from getting-started.html does not work.

### ISSUE THREAD

**Issue by Kasheftin** [sebastianbergmann/phpunit#2485]
Hi,
I just opened the https://phpunit.de/getting-started.html page and pasted instructions from there. I have PHP 7.0.8 and PHPUnit 6.0.5. I created files `email.php` and `emailTest.php` and run `phpunit --bootstrap email.php emailTest` and got:

```
$ phpunit --bootstrap email.php emailTest
PHPUnit 6.0.5 by Sebastian Bergmann and contributors.

E.E                                                                 3 / 3 (100%)

Time: 251 ms, Memory: 8.00MB

There were 2 errors:

1) EmailTest::testCanBeCreatedFromValidEmailAddress
TypeError: Return value of Email::ensureIsValidEmail() must be an instance of void, none returned

/home/www/tevai/api/engine/test/email.php:35
/home/www/tevai/api/engine/test/email.php:10
/home/www/tevai/api/engine/test/email.php:17
/home/www/tevai/api/engine/test/emailTest.php:15

2) EmailTest::testCanBeUsedAsString
TypeError: Return value of Email::ensureIsValidEmail() must be an instance of void, none returned

/home/www/tevai/api/engine/test/email.php:35
/home/www/tevai/api/engine/test/email.php:10
/home/www/tevai/api/engine/test/email.php:17
/home/www/tevai/api/engine/test/emailTest.php:30

ERRORS!
Tests: 3, Assertions: 1, Errors: 2.
```

What should I do next?

**Comment by sebastianbergmann** [comment]
This [should](https://github.com/sebastianbergmann/phpunit/blob/master/CONTRIBUTING.md#reporting-issues) have been reported on the [phpunit-documentation](https://github.com/sebastianbergmann/phpunit-website) project.

That being said, the example in question uses PHP 7.1 syntax and is not compatible with PHP 7.0.

**Comment by sebastianbergmann** [comment]
The example on https://phpunit.de/getting-started.html is now available for PHP 7.1, PHP 7.0, and PHP 5.6.
